File: DuDoong-Common/src/main/java/band/gosrock/common/dto/OIDCDecodePayload.java
package band.gosrock.common.dto;


import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class OIDCDecodePayload {
    /** issuer ex https://kauth.kakao.com */
    private String iss;
    /** client id */
    private String aud;
    /** oauth provider account unique id */
    private String sub;

    private String email;
}

File: DuDoong-Common/src/main/kotlin/band/gosrock/common/dto/OIDCDecodePayload.kt
package band.gosrock.common.dto

data class OIDCDecodePayload(
    /** issuer ex https://kauth.kakao.com */
    val iss: String,
    /** client id */
    val aud: String,
    /** oauth provider account unique id */
    val sub: String,
    val email: String,
)
